CheatSheet/scripts/setup-supabase.sh

```
@@ -0,0 +1,156 @@
+    else
+        echo -e "${RED}❌ Unsupported OS. Please install Supabase CLI manually:${NC}"
+        echo "https://supabase.com/docs/guides/cli"
+        exit 1
+    fi
+fi
+
+echo -e "${GREEN}✅ Supabase CLI found${NC}"
+
+# Load environment variables
+if [ -f "../.env" ]; then
+    export $(cat ../.env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)